app/controllers/concerns/leaderboard_controller.rb

```diff
@@ -0,0 +1,14 @@
+module LeaderboardController
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :find_scores, only: :show
+  end
+
+  def show
+  end
```

Comment:
Because I must define a method for the *show* action. The method is empty because Rails will render the controller's `show.html.erb` template by default and the `#find_scores` before filter initializes `@scores` (formerly `@leaderboard`).